doc/classes/SpinBox.xml

```diff
@@ -5,6 +5,16 @@
 	</brief_description>
 	<description>
 		SpinBox is a numerical input text field. It allows entering integers and floats.
+		[b]EXAMPLE:[/b]
```

Comment:
To my knowledge, we don't use uppercase for emphasis in the documentation anywhere else, so I think this makes more sense:

```suggestion
		[b]Example:[/b]
```

Reply:
> Bold text is now supported in the editor help, so it's a good idea to use it above example codeblocks. 👍

Cool!

Reply:
@Calinou, You think you could add Color tags also?

Reply:
@zaksnet I just said it was a possibility, but it's quite involved. It basically involves parsing GDScript code blocks when generating the editor help, inserting placeholder color codes (e.g. `[color={variable_color}]`), then replacing those placeholders to follow the current script editor syntax theme.

Reply:
Sorry, i was talking about `color={variable_color}]` so that i could possibly color a single word, not an entire syntax highlight. Anyway, I hadn't noticed this was implemented already, thanks!